packages/venia-concept/venia-upward.yml

```diff
@@ -7,6 +7,9 @@ response:
     - matches: request.url.pathname
       pattern: '^/(graphql|rest|media/)'
       use: proxy
+    - matches: request.url.pathname
```

Comment:
It's exciting to see someone else writing UPWARD configuration! Thank you for diving in.